code/__HELPERS/_logging.dm

```diff
@@ -129,3 +129,8 @@
 		return "[A.loc] [COORD(T)] ([A.loc.type])"
 	else if(A.loc)
 		return "[A.loc] (0, 0, 0) ([A.loc.type])"
+
+
+/proc/log_manifest(key,datum/mind/mind,mob/body,latejoin = FALSE)
+	if (CONFIG_GET(flag/log_manifest))
+		WRITE_FILE(GLOB.manifest_log, "[key] \\ [body.real_name] \\ [mind.assigned_role] \\ [mind.special_role ? mind.special_role : "NONE"] \\ [latejoin ? "LATEJOIN":"ROUNDSTART"]")
```

Comment:
For the sake of consistency, can we use a | instead of /?

Reply:
I specifically didn't use it because AI's can use it in their names

Reply:
Oh! Never mind then :)

Reply:
why can AIs use | anyway? i don't think i've ever seen an AI use it actually